cirq/google/xmon_device_test.py

```diff
@@ -227,31 +227,6 @@ def test_validate_scheduled_operation_not_adjacent_exp_11_exp_w():
     d.validate_schedule(s)
 
 
-def test_validate_circuit_repeat_measurement_keys():
```

Comment:
Why was this test removed with no test being added to e.g. circuit_test.py to compensate? What is checking that we detect dupes?